ONSdigital/ras-frontstage
# Privacy update

# Motivation and Context
Update secure message form with clarification on personal data, linked to /my-account


# What has changed
Minor change to secure messaging form




# How to test?
Deploy to dev cluster. Go to "To-do" and click "Send a message". The form should now match the mockup in the story [Privacy notification update to secure messaging platform](https://collaborate2.ons.gov.uk/confluence/pages/viewpage.action?pageId=42295192#Useraccountcreation(RAS/RM)-RAS-235), and clicking the link in the new sentence should take you to the Account Details form.




# Links
- https://trello.com/c/oO3qMudG
- [Privacy notification update to secure messaging platform](https://collaborate2.ons.gov.uk/confluence/pages/viewpage.action?pageId=42295192#Useraccountcreation(RAS/RM)-RAS-235)




# Screenshots (if appropriate):
